api/src/main/java/javax/enterprise/inject/spi/AfterTypeDiscovery.java

```diff
@@ -76,4 +78,29 @@
      */
     public void addAnnotatedType(AnnotatedType<?> type, String id);
 
+    /**
+     * <p>
+     * Obtains a new {@link AnnotatedTypeConfigurator} to configure a new {@link javax.enterprise.inject.spi.AnnotatedType} and
+     * add it to the set of types which will be scanned during bean discovery at the end of the observer invocation
+     * </p>
+     *
+     * <p>
+     * This method allows multiple annotated types, based on the same underlying type, to be defined with a builder.
```

Comment:
`to be defined with a builder`? This is probably a leftover of the previous version.